index.html

```diff
@@ -120,7 +130,128 @@ <h2>Introduction</h2>
       <p class="issue" data-number="7"></p>
     </section>
 
-  <section id="conformance"></section>
+    <section id="conventions">
+      <h2>Documentation Conventions</h2>
+      <p>
+        This document uses the same conventions as [[TTML2]] for the specification of parameter attributes, styling attributes and metadata elements. In particular:
+      </p>
+      <p>
+        Section 2.3 of [[TTML2]] specifies conventions used in the [[XML]] representation of elements; and
+        Sections 6.2 and 8.2 of [[TTML2]] specify conventions used when specifying the syntax of attribute values.
+      </p>
+      <p>
+        All content of this specification that is not explicitly marked as non-normative is considered to be normative.
+        If a section or appendix header contains the expression "non-normative", then the entirety of the section or appendix is considered non-normative.
+      </p>
+      <p>
+        This specification uses Feature designations as defined in Appendices E at [[TTML2]]:
+        when making reference to content conformance, these designations refer to the syntactic expression or the semantic capability associated with each designated Feature; and
+        when making reference to processor conformance, these designations refer to processing requirements associated with each designated Feature.
+        If the name of an element referenced in this specification is not namespace qualified, then the TT namespace applies (see <a href="#namespaces">9.3 Namespaces</a>.)
+      </p>
```

Comment:
Is this really a "document convention"? It would seem more appropriate in the conformance section.

Reply:
I'm not sure it makes much practical difference?

Reply:
It does I think since we can then have a discussion on conformance in one place.